plugins/services/tasks/local.go

```diff
@@ -62,8 +62,14 @@ import (
 )
 
 var (
-	_     = (api.TasksClient)(&local{})
-	empty = &ptypes.Empty{}
+	_                    = (api.TasksClient)(&local{})
+	empty                = &ptypes.Empty{}
+	tasksServiceRequires = []plugin.Type{
+		plugins.EventPlugin,
```

Comment:
`plugins.EventPlugin` was not previously enabled on Darwin.  I don't have a Mac to test on; @dmcgowan would this cause any problems?

Reply:
I manually tested it and containerd starts up properly. Also note that FreeBSD had (and still has) the same setup.

Reply:
Yeah this is just the requires, shouldn't be a problem since require just ensures the plugin graph is properly ordered.